collectors/feature/jira/src/main/java/com/capitalone/dashboard/client/team/TeamDataClientImpl.java

```diff
@@ -63,13 +60,6 @@ public int updateTeamInformation() {
 		if (CollectionUtils.isNotEmpty(teams)) {
 			updateMongoInfo(teams);
 			count += teams.size();
-		} else {
-			List<BasicProject> projects = jiraClient.getProjects();
-			
-			if (CollectionUtils.isNotEmpty(projects)) {
-				updateMongoInfoLegacy(projects);
-				count += projects.size();
-			}
```

Comment:
Is it correct to say that the feature documents will NOT link back to a collector item? I wouldn't want  (projectA, teamX), (projectA, Any), (Any, teamX) to result in a bunch of duplicated features. The git collector suffers from this problem - there are a lot of duplicate commits because they are separated out by collector_item. In our case since we query on-demand based on parameters instead of by ids we can share the data.